src/TouringPanel.ts

```diff
@@ -63,9 +63,18 @@ class TouringPanel {
 
   private addEventListeners() {
     this.events.on(PanelTabEvents.SHOW_PANEL, () => {
+      if (this.active === true) {
+        return; //Don't update tasks when clicking on open touring button and touring panel is already open
```

Comment:
Spaces

```suggestion
        return; // Don't update tasks when clicking on open touring button and touring panel is already open
```

Please check other comments too.